mestopy/mestopy.py

```diff
@@ -1 +1,242 @@
-"""Main module."""
+import numpy as np
+from pyfar import Signal
+
+
+# Class to generate ref-Objects, that can bei part of the MeasurementChain
+class DeviceObj(object):
+    """Class for device in MeasurementChain.
+
+    This class holds methods and properties of a device in the
+    'MeasurementChain' class.
```

Comment:
Maybe add an example, e.g., a sound card or pre-amplifier for an easier understanding